doc/LocalVMSetup.md

```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
-Please refer to: https://fedoraproject.org/wiki/EPEL
+## 4. Run the following commands.
+
+$ sudo yum install epel-release
```

Comment:
Use [Markdown formatting](https://arcticicestudio.github.io/styleguide-markdown/rules/code.html#no-shell-code-dollar-sign) for commands:

```bash
sudo yum install epel-release
```